src/rpm/postinstall.sh

```diff
@@ -42,8 +35,6 @@ if [ "$(id -u)" -ne 0 ]; then
     exit 1
 fi
 
-microshift_config
```

Comment:
Did we disabled the lvms in 'community' builds? I don't see in it downstream repo

Reply:
It is disabled automatically when topolvm is installed (see [this code](https://github.com/microshift-io/microshift/blob/main/src/topolvm/topolvm.spec#L59)).
It now occured to me that we need to "always" disable it. I will open a bug.